package.json

```diff
@@ -53,6 +53,7 @@
 		"clean:nyc": "rimraf --glob \"nyc/**\"",
 		"clean:r11s": "fluid-build -g server --task clean",
 		"clean:root": "rimraf --glob \"*.done.build.log\"",
+		"code-coverage-analysis:report": "flub report codeCoverage",
```

Comment:
```suggestion
		"codeCoverage:report": "flub report codeCoverage",
```

Reply:
Is there a "collect" step like there is for bundle size?

Reply:
The tests create the coverage stats when run with coverage command and the Publish step/task in yml, publish those into artifacts.